e2e-tests/functions

```diff
@@ -116,22 +116,40 @@ get_cr() {
 	cat "${TEMP_DIR}/cr.yaml"
 }
 
-run_psql() {
+run_psql_local() {
```

Comment:
For me it's a bit strange that we call it `local` but we require uri/host, local would suggest localhost/pod (to me at least).
I would suggest maybe to call it `direct` instead and the other one `proxy` or `pgbouncer`. In that case it would be clear from where it is executing the query.
But this is just a clarity suggestion so I leave it up to you.

Reply:
up to @nmarukovich but maybe we can do it later

Reply:
yep. let's do it in next PR with tests. I will do it.